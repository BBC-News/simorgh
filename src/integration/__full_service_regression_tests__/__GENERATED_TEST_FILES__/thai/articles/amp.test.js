/**
 * @service thai
 * @pathname /thai/articles/c442rl3md0eo
 *
 * THIS TEST SUITE WAS GENERATED BY A BUILD SCRIPT. DO NOT EDIT THIS FILE.
 */

import runCrossPlatformTests from '../../../../pages/articles/crossPlatformTests';
import runAmpTests from '../../../../pages/articles/ampTests';
describe(platform.toUpperCase(), () => {
  describe(pageType, () => {
    runCrossPlatformTests();
    runAmpTests();
  });
});
