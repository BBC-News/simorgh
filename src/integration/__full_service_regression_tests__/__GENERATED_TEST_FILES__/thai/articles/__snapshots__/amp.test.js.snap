// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AMP  A11y I can see the skip to content link 1`] = `"ข้ามไปยังเนื้อหา"`;

exports[`AMP  AMP Consent config 1`] = `
Object {
  "consents": Object {
    "user-consent": Object {
      "promptIfUnknownForGeoGroup": "eea",
      "promptUI": "consent-prompt",
    },
  },
}
`;

exports[`AMP  AMP Geo config 1`] = `
Object {
  "ISOCountryGroups": Object {
    "eea": Array [
      "at",
      "ax",
      "be",
      "bg",
      "bl",
      "cy",
      "cz",
      "de",
      "dk",
      "ea",
      "ee",
      "es",
      "fi",
      "fr",
      "gb",
      "gf",
      "gg",
      "gi",
      "gp",
      "gr",
      "hr",
      "hu",
      "ic",
      "ie",
      "im",
      "is",
      "it",
      "je",
      "li",
      "lt",
      "lu",
      "lv",
      "mf",
      "mq",
      "mt",
      "nc",
      "nl",
      "no",
      "pf",
      "pl",
      "pm",
      "pt",
      "re",
      "ro",
      "se",
      "si",
      "sj",
      "sk",
      "tf",
      "va",
      "wf",
      "yt",
    ],
  },
}
`;

exports[`AMP  Footer I can click on the BBC branding and it would take me to the homepage 1`] = `"/thai"`;

exports[`AMP  Footer I can see the footer copyright and external linking text 1`] = `"© 2020 บีบีซี. บีบีซีไม่มีส่วนรับผิดชอบต่อเนื้อหาของเว็บไซต์ภายนอก. นโยบายของเราเรื่องการเชื่อมต่อไปยังลิงก์ภายนอก."`;

exports[`AMP  Header I can see the branding 1`] = `"BBC News, ไทย"`;

exports[`AMP  I can see an image with a caption 1`] = `"https://ichef.bbci.co.uk/news/640/cpsdevpb/d624/test/fc3ebc60-1105-11ea-acaa-cd607a38aea4.jpg"`;

exports[`AMP  I can see an image with a caption 2`] = `"คำบรรยายภาพ ภาพทดสอบลิขสิทธิ์บีบีซีนี้แสดงแผนที่ของฝรั่งเศส รูปภาพอยู่ในสามบล็อกแรกและมีคำบรรยายภาพนี้"`;

exports[`AMP  SEO Canonical link 1`] = `"http://localhost:7080/thai/articles/c442rl3md0eo"`;

exports[`AMP  SEO Dir attribute 1`] = `"ltr"`;

exports[`AMP  SEO FB admins 1`] = `"100004154058350"`;

exports[`AMP  SEO FB app ID 1`] = `"1609039196070050"`;

exports[`AMP  SEO Heading level 1 1`] = `"นี่คือหัวข้อข่าวของบทความทดสอบนี้"`;

exports[`AMP  SEO Lang attribute 1`] = `"th"`;

exports[`AMP  SEO Linked data 1`] = `
Object {
  "@context": "http://schema.org",
  "@type": "Article",
  "author": Object {
    "@type": "NewsMediaOrganization",
    "logo": Object {
      "@type": "ImageObject",
      "height": 576,
      "url": "https://news.files.bbci.co.uk/ws/img/logos/og/thai.png",
      "width": 1024,
    },
    "name": "BBC News ไทย",
    "noBylinesPolicy": "https://www.bbc.com/thai/institutional-49281839#authorexpertise",
  },
  "dateModified": "2019-11-27T11:14:47.993Z",
  "datePublished": "2019-11-27T11:14:47.993Z",
  "headline": "นี่คือหัวข้อข่าวของบทความทดสอบนี้",
  "image": Object {
    "@type": "ImageObject",
    "height": 576,
    "url": "https://news.files.bbci.co.uk/ws/img/logos/og/thai.png",
    "width": 1024,
  },
  "mainEntityOfPage": Object {
    "@id": "http://localhost:7080/thai/articles/c442rl3md0eo",
    "@type": "WebPage",
    "name": "นี่คือหัวข้อข่าวของบทความทดสอบนี้",
  },
  "publisher": Object {
    "@type": "NewsMediaOrganization",
    "logo": Object {
      "@type": "ImageObject",
      "height": 576,
      "url": "https://news.files.bbci.co.uk/ws/img/logos/og/thai.png",
      "width": 1024,
    },
    "name": "BBC News ไทย",
    "publishingPrinciples": "https://www.bbc.com/thai/institutional-49281839",
  },
  "thumbnailUrl": "https://news.files.bbci.co.uk/ws/img/logos/og/thai.png",
  "url": "http://localhost:7080/thai/articles/c442rl3md0eo",
}
`;

exports[`AMP  SEO OG description 1`] = `"นี่คือหัวข้อข่าวของบทความทดสอบนี้"`;

exports[`AMP  SEO OG image 1`] = `"https://news.files.bbci.co.uk/ws/img/logos/og/thai.png"`;

exports[`AMP  SEO OG image alt 1`] = `"BBC News ไทย"`;

exports[`AMP  SEO OG locale 1`] = `"th-TH"`;

exports[`AMP  SEO OG site name 1`] = `"BBC News ไทย"`;

exports[`AMP  SEO OG title 1`] = `"นี่คือหัวข้อข่าวของบทความทดสอบนี้ - BBC News ไทย"`;

exports[`AMP  SEO OG type 1`] = `"article"`;

exports[`AMP  SEO OG url 1`] = `"http://localhost:7080/thai/articles/c442rl3md0eo"`;

exports[`AMP  SEO Page title 1`] = `"นี่คือหัวข้อข่าวของบทความทดสอบนี้ - BBC News ไทย"`;

exports[`AMP  SEO Robots meta tag 1`] = `"noodp,noydir"`;

exports[`AMP  SEO Twitter card 1`] = `"summary_large_image"`;

exports[`AMP  SEO Twitter creator 1`] = `"@bbc_thailand"`;

exports[`AMP  SEO Twitter description 1`] = `"นี่คือหัวข้อข่าวของบทความทดสอบนี้"`;

exports[`AMP  SEO Twitter image alt 1`] = `"BBC News ไทย"`;

exports[`AMP  SEO Twitter image src 1`] = `"https://news.files.bbci.co.uk/ws/img/logos/og/thai.png"`;

exports[`AMP  SEO Twitter site 1`] = `"@bbc_thailand"`;

exports[`AMP  SEO Twitter title 1`] = `"นี่คือหัวข้อข่าวของบทความทดสอบนี้ - BBC News ไทย"`;
